src/bcf/record.rs

```diff
@@ -140,6 +166,26 @@ impl Record {
         (0..n).map(|i| unsafe { ffi::CStr::from_ptr(alleles[i]).to_bytes() }).collect()
     }
 
+    /// Set alleles. Alleles can be given as anything that implements `AsRef<[u8]>`, i.e.,
+    /// can be converted into a slice of `u8`. For example, a vector of slices, vector of vectors,
+    /// slices of slices, and so on.
+    pub fn update_alleles<'a, A: AsRef<[u8]>>(&mut self, alleles: &'a [A]) -> Result<(), AlleleWriteError>
```

Comment:
Can you also rename this to `set_alleles`?

Reply:
Done.